src/Components/Filters/Filters.css

```diff
@@ -0,0 +1,32 @@
+.Filters {
+  flex-basis: 40%;
+}
+
+.filter-block {
+  font-size: 1em;
+  text-align: center;
+  border: gray solid 1px;
+  padding: 20px;
+  margin: 4px;
+}
+
+#filter1 {
```

Comment:
Recommendation! 
You can use `#id` but you shouldn't. Think twice before using id. Every time you want to use id, at first try to use the class name.